README.md

```diff
@@ -0,0 +1,42 @@
+# Blogger
```

Comment:
The first thing in your README should be the necessary import statement to use your library!

Reply:
Oh, I did not see it as a library. Perhaps I will look around for a sample library Readme.

Reply:
In Go, everything should be a library!

Reply:
well noted.